test/rules/prefer-conditional-expression/default/test.ts.lint

```diff
@@ -50,4 +50,33 @@ if (true) {
     foo(bar).baz = 1;
 }
 
+// checks return statements
+if (true) {
+~~ [err2]
+    return 1;
+} else {
+    return 2;
+}
+
+if (true) {
+    return 1;
+} else if (true) {
+       ~~ [err2]
```

Comment:
without `"check-else-if"` there should be no error on `if-elseif-else` statements